### ISSUE TITLE
+ in monitors list is very small

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-gui#2158]
### Expected Behavior

'+' should be more hittable

### Actual Behavior

'+' is very tiny and might not be obvious that it's a button
We have seen issues with people having trouble moving the mouse very accurately so it may be hard to click
![image](https://user-images.githubusercontent.com/2855464/40555623-b579a376-6017-11e8-859b-c346622d2d15.png)

### Steps to Reproduce

1. Make a monitor list

### Operating System and Browser

Windows 10 Firefox

**Comment by sjhuang26** [comment]
Considering how Scratch 2 lists are done, I think we should add a circle around the "+" sign.
![image](https://user-images.githubusercontent.com/29558846/40570375-276a3410-6058-11e8-9614-5793527b10b6.png)

It might also be nice to clarify that the corner is draggable with a symbol clearer than "=".

**Comment by paulkaplan** [comment]
@sjhuang26 yes, I didn't have time to get those icons in before this first iteration went out, but if you want to make a PR for bringing the design up-to-date with Scratch2 that'd be good, then @carljbowman and I can re-evaluate sizing and touch after.
